src/components/Social.jsx

```diff
@@ -27,7 +28,7 @@ class Social extends React.Component {
 
   render() {
     return <div>
-      <Location>{'Project We Love'}{`    `}{'Food and Drink'} {this.state.location} {'Project Budget'}</Location>
+      <Location>{'Project We Love'}{`    `}{this.state.category} {this.state.location} {'Project Budget'}</Location>
```

Comment:
Here you've remained consistent with inherited language and used "Project we love" instead of "product we love" elsewhere.